### ISSUE TITLE
Documentation Version History

### ISSUE THREAD

**Issue by SueyChop** [pymeasure/pymeasure#690]
The Read the Docs documentation does not include a version history or changelog, but would be useful in my opinion.

**Comment by bilderbuchi** [comment]
Agreed. This probably fits best as a page in the "About pymeasure" section. As the file is already ReST, it should be easy to integrate if modern sphinx can easily integrate files outside of the `docs/` tree.

**Comment by maederan201** [comment]
I think this has been addressed in the past (though I could not find a PR for it...) and could  therefore be closed, right?

**Comment by BenediktBurger** [comment]
You are right, the changelog is on read the docs: https://pymeasure.readthedocs.io/en/latest/about/changes.html .

Thanks for pointing out.
